src/Hedgehog/Gen.fs

```diff
@@ -354,7 +368,7 @@ module Gen =
 
     /// Generates a random string.
     let string : Gen<string> =
-        choice [alphaNum; charBounded] |> string'
+        choice [alphaNum; unicodeAll] |> string'
```

Comment:
Hmm... True, `unicode` is better default, I'll change it.

And, I think, we need something similar in the Haskell version? Right now it uses min/max bounds of char, no?